File: Java/Class_basic/Animal.java
package Class_basic;

import java.util.Scanner;

class Animal {
    private String name;
    private int age;
    private double height, weight;
    private String sex;

    public String getName() {
        return name;
    }

    public void setName(String value) {
        name = value;
    }

    public int getAge() {
        return age;
    }

    public void setAge(int value) {
        age = value;
    }

    public double getHeight() {
        return height;
    }

    public void setHeight(double value) {
        height = value;
    }

    public double getWeight() {
        return weight;
    }

    public void setWeight(double value) {
        weight = value;
    }

    public String getSex() {
        return sex;
    }

    public void setSex(String value) {
        sex = value;
    }

    public Animal() {
        name = "PET";
        age = 0;
        height = weight = 0.0;
        sex = "Unknown";
    }

    public Animal(String name, int age, double height, double weight, String sex) {
        this.name = name;
        this.age = age;
        this.height = height;
        this.weight = weight;
        this.sex = sex;
    }

    public Animal(Animal pet) {
        this.name = pet.name;
        this.age = pet.age;
        this.height = pet.height;
        this.weight = pet.weight;
        this.sex = pet.sex;
    }

    public void Input() {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Nhap ten pet: ");
        name = scanner.nextLine();
        System.out.print("Nhap tuoi: ");
        age = Integer.parseInt(scanner.nextLine());
        System.out.print("Nhap chieu cao: ");
        height = Double.parseDouble(scanner.nextLine().replace(',', '.'));
        System.out.print("Nhap can nang: ");
        weight = Double.parseDouble(scanner.nextLine().replace(',', '.'));
        System.out.print("Nhap gioi tinh: ");
        sex = scanner.nextLine();
    }

    public void Output() {
        System.out.println("Pet Info: ");
        System.out.println("Name: " + name);
        System.out.println("Age: " + age);
        System.out.printf("Height: %.2f%n", height);
        System.out.printf("Weight: %.2f%n", weight);
        System.out.println("Sex: " + sex);
    }

    public String showName() {
        return this.name;
    }

    public int showAge() {
        return this.age;
    }

    public void move() {
        System.out.println("Animal is moving...");
    }

    public void sleep() {
        System.out.println("Animal is sleeping.");
    }
}

File: Theory_Classroom/C#/Class_basic/Animal.cs
using System;

namespace Class_basic{
    class Animal{
        string name, sex;
        int age;
        double height, weight;
        // get - set
        public string name_{
            get { return name; }
            set { name = value; }
        }
        public int age_{
            get { return age; }
            set { age = value; }
        }
        public double height_{
            get { return height; }
            set { height = value; }
        }
        public double weight_{
            get { return weight; }
            set { weight = value; }
        }
        public string sex_{
            get { return sex; }
            set { sex = value; }
        }
        // Phương thức khởi tạo
        public Animal(){
            name = "PET";
            age = 0;
            height = weight = 0.0;
            sex = "Unknown";
        }
        public Animal(string name, int age, double height, double weight, string sex){
            this.name = name;
            this.age = age;
            this.height = height;
            this.weight = weight;
            this.sex = sex;
        }
        public Animal(Animal pet){
            this.name = pet.name;
            this.age = pet.age;
            this.height = pet.height;
            this.weight = pet.weight;
            this.sex = pet.sex;
        }
        // Phương thức xử lý
        public void Input(){
            Console.Write("Nhap ten pet: ");
            name = Console.ReadLine();
            Console.Write("Nhap tuoi: ");
            age = int.Parse(Console.ReadLine());
            Console.Write("Nhap chieu cao: ");
            height = double.Parse(Console.ReadLine().Replace('.',','));
            Console.Write("Nhap can nang: ");
            weight = double.Parse(Console.ReadLine().Replace('.',','));
            Console.Write("Nhap gioi tinh: ");
            sex = Console.ReadLine();
        }
        public void Output(){
            Console.WriteLine("Pet Info: ");
            Console.WriteLine("Name: " + name);
            Console.WriteLine("Age: " + age);
            Console.WriteLine("Height: {0:0.00}", height);
            Console.WriteLine("Weight: {0:0.00}", weight);
            Console.WriteLine("Sex: " + sex);
        }
        public string showName(){
            return this.name;
        }
        public int showAge(){
            return this.age;
        }
        public void move(){
            Console.WriteLine("Animal is moving...");
        }
        public void sleep(){
            Console.WriteLine("Animal is sleeping.");
        }
    }
}